core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -36,7 +39,7 @@ oppia.directive('simpleEditorSidebar', [
             'Multiple choice', 'Correct answer', 'Hints', 'Bridge text'];
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
-
+          $scope.modeSrv = SimpleEditorSidebarModeService;
```

Comment:
Yeah, I realize it's long. You could call it $scope.SidebarModeService perhaps, but I'm OK with using the long name too.